src/20_Components/amp-3d-gltf.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  ## Introduction
+
+  The amp-3d-gltf component displays 3D models that are in glTF format.
```

Comment:
Can you please link `amp-3d-gltf` to the official docs.

Reply:
done